### PR TITLE
Eslint fixes for multiline-comment-style

### PR DESCRIPTION
**Pull request by reosarevok**
Mostly autofixes, but went through it to make a few additional changes: fix too long lines, start with caps when I noticed them missing (no doubt I missed a few) and make unnecessarily multiline comments single-line.

In separate commits, replaced some long copyright blurbs by the shorter version and some (c) by (C).

I will now stop sending huge eslint patches for a while :p

### CODE REVIEW

File: root/static/scripts/edit/forms.js [hunk 1]
```diff
@@ -244,30 +262,31 @@ ko.bindingHandlers.loop = {
...
+ * concatenated with the index of the item in the loop.
+ *
+ * So if you have something like this in the template:
+ *
+ * <!-- ko foreach: items -->
+ * <tr>
+ * <td><label>Foo</label></td>
+ * <td><input data-bind="withLabel: 'foo'" /></td>
+ * <tr>
+ * <!-- /ko -->
```
Comment (mwiencek) [inline root/static/scripts/edit/forms.js]: The indentation here was messed up compared to the previous version.

File: root/static/scripts/edit/MB/CoverArt.js [hunk 2]
```diff
@@ -264,8 +257,10 @@ MB.CoverArt.upload_image = function (postfields, file) {
     /* IE10 and older don't have overrideMimeType. */
     if (typeof(xhr.overrideMimeType) === 'function')
     {
-        /* prevent firefox from parsing a 204 No Content response as XML.
-           https://bugzilla.mozilla.org/show_bug.cgi?id=884693 */
+        /*
+         * prevent firefox from parsing a 204 No Content response as XML.
+         *https://bugzilla.mozilla.org/show_bug.cgi?id=884693
```
Comment (mwiencek) [inline root/static/scripts/edit/MB/CoverArt.js]:
```suggestion
         * https://bugzilla.mozilla.org/show_bug.cgi?id=884693
```

File: root/static/scripts/edit/MB/CoverArt.js [hunk 3]
```diff
@@ -378,8 +373,10 @@ MB.CoverArt.FileUpload = function (file) {
 
         if (self.status() === 'done' || self.busy())
         {
-            /* This file is currently being uploaded or has already
-               been uploaded. */
+            /*
+             * This file is currently being uploaded or has already
+             *been uploaded.
```
Comment (mwiencek) [inline root/static/scripts/edit/MB/CoverArt.js]:
```suggestion
             * been uploaded.
```

File: root/static/scripts/edit/MB/CoverArt.js [hunk 4]
```diff
@@ -168,8 +159,10 @@ MB.CoverArt.validate_file = function (file) {
     reader.addEventListener("loadend", function () {
         var uint32view = new Uint32Array(reader.result);
 
-        /* JPEG signature is usually FF D8 FF E0 (JFIF), or FF D8 FF E1 (EXIF).
-           Some cameras and phones write a different fourth byte. */
+        /*
+         * JPEG signature is usually FF D8 FF E0 (JFIF), or FF D8 FF E1 (EXIF).
+         *Some cameras and phones write a different fourth byte.
```
Comment (mwiencek) [inline root/static/scripts/edit/MB/CoverArt.js]:
```suggestion
         * Some cameras and phones write a different fourth byte. 
```

File: root/static/scripts/edit/MB/CoverArt.js [hunk 5]
```diff
@@ -58,10 +45,12 @@ MB.CoverArt.image_error = function ($img, image) {
...
     {
-        /* image doesn't exist at all, perhaps it was removed
-           between requesting the index and loading the image.
-           FIXME: start over if this happens?  obviously the
-           data in the index is incorrect. */
+        /*
+         * image doesn't exist at all, perhaps it was removed
+         * between requesting the index and loading the image.
+         * FIXME: start over if this happens?  obviously the
+         *data in the index is incorrect.
```
Comment (mwiencek) [inline root/static/scripts/edit/MB/CoverArt.js]:
```suggestion
         * data in the index is incorrect. 
```

File: root/static/scripts/common/MB/Control/Autocomplete.js [hunk 6]
```diff
@@ -980,27 +1012,27 @@ function renderContainingAreas(area) {
...
+ * <img class="search" src="search.png" />
+ * <input type="text" class="name" />
+ * <input type="hidden" class="id" />
+ * <input type="hidden" class="gid" />
+ * </span>
+ *
+ * Do a lookup of the span with jQuery and pass it into EntityAutocomplete
+ * as options.inputs, for example, for a release group do this:
+ *
+ * MB.Control.EntityAutocomplete({ inputs: $('span.release-group.autocomplete') });
```
Comment (yvanzo) [inline root/static/scripts/common/MB/Control/Autocomplete.js]: Indentation has been incidentally removed for both examples.

File: root/static/scripts/common/MB/Control/Autocomplete.js [hunk 7]
```diff
@@ -179,8 +191,10 @@ $.widget("mb.entitylookup", $.ui.autocomplete, {
             var recent = self.recentEntities();
 
             if (!this.value && recent && recent.length && !self.menu.active) {
-                // setting ac.term to "" prevents the autocomplete plugin
-                // from running its own search, which closes our menu.
+                /*
+                 * setting ac.term to "" prevents the autocomplete plugin
```
Comment (yvanzo) [inline root/static/scripts/common/MB/Control/Autocomplete.js]:
```suggestion
                 * Setting ac.term to "" prevents the autocomplete plugin
```

File: root/static/scripts/common/MB/Control/Menu.js [hunk 8]
```diff
@@ -49,8 +36,10 @@ MB.Control.HeaderMenu = function () {
     });
 
     $('body').on('click', function (event) {
-        // clicks outside of the menu (anything that reaches the body) should
-        // close the menu
+        /*
+         * clicks outside of the menu (anything that reaches the body) should
```
Comment (yvanzo) [inline root/static/scripts/common/MB/Control/Menu.js]:
```suggestion
         * Clicks outside of the menu (anything that reaches the body) should
```

File: root/static/scripts/common/MB/Control/Menu.js [hunk 9]
```diff
@@ -49,8 +36,10 @@ MB.Control.HeaderMenu = function () {
...
+         * close the menu
```
Comment (yvanzo) [inline root/static/scripts/common/MB/Control/Menu.js]:
```suggestion
         * close the menu.
```

File: root/static/scripts/edit/MB/Control/ArtistEdit.js [hunk 10]
```diff
@@ -31,14 +33,15 @@ MB.Control.ArtistEdit = function () {
...
-           Choir: 6
-    */
+    /*
+     * Sets the label descriptions depending upon the artist type:
+     *
+     * Unknown: 0
+     * Person: 1
+     * Group: 2
+     * Orchestra: 5
+     * Choir: 6
```
Comment (yvanzo) [inline root/static/scripts/edit/MB/Control/ArtistEdit.js]: Indentation has been incidentally removed.

File: root/static/scripts/edit/MB/CoverArt.js [hunk 11]
```diff
@@ -114,9 +103,11 @@ MB.CoverArt.reorder_position = function () {
         }
     );
 
-    /* moving <script> elements around with insertBefore() and
+    /*
+     * moving <script> elements around with insertBefore() and
```
Comment (yvanzo) [inline root/static/scripts/edit/MB/CoverArt.js]:
```suggestion
     * Moving <script> elements around with insertBefore() and
```

File: root/static/scripts/edit/MB/CoverArt.js [hunk 12]
```diff
@@ -264,8 +257,10 @@ MB.CoverArt.upload_image = function (postfields, file) {
     /* IE10 and older don't have overrideMimeType. */
     if (typeof (xhr.overrideMimeType) === 'function')
     {
-        /* prevent firefox from parsing a 204 No Content response as XML.
-           https://bugzilla.mozilla.org/show_bug.cgi?id=884693 */
+        /*
+         * prevent firefox from parsing a 204 No Content response as XML.
```
Comment (yvanzo) [inline root/static/scripts/edit/MB/CoverArt.js]:
```suggestion
         * Prevent firefox from parsing a 204 No Content response as XML.
```

File: root/static/scripts/guess-case/MB/GuessCase/Handler/Base.js [hunk 13]
```diff
@@ -686,8 +673,8 @@ MB.GuessCase.Handler.Base = function (gc) {
         }
 
         if (tmp.length > 2) {
-            var s = tmp.join(""); // yes,we have an acronym, get string
-            s = s.replace(/(\.)*$/,"."); // replace any number of trailing "." with ". "
+            var s = tmp.join(""); // Yes, we have an acronym, get string
+            s = s.replace(/(\.)*$/,"."); // Feplace any number of trailing "." with ". "
```
Comment (yvanzo) [inline root/static/scripts/guess-case/MB/GuessCase/Handler/Base.js]:
```suggestion
            s = s.replace(/(\.)*$/,"."); // Covfefeplace any number of trailing "." with ". "
```

File: root/static/scripts/guess-case/MB/GuessCase/Input.js [hunk 14]
```diff
@@ -239,8 +198,10 @@ MB.GuessCase.Input = function (gc) {
         }
         for (var i = 0; i < chars.length; i++) {
             if (chars[i].match(gc.re.SPLITWORDSANDPUNCTUATION)) {
-                // see http://www.codingforums.com/archive/index.php/t-49001
-                // for reference (escaping the sequence)
+                /*
+                 * see http://www.codingforums.com/archive/index.php/t-49001
```
Comment (yvanzo) [inline root/static/scripts/guess-case/MB/GuessCase/Input.js]:
```suggestion
                 * See http://www.codingforums.com/archive/index.php/t-49001
```

File: root/static/scripts/release-editor/trackParser.js [hunk 15]
```diff
@@ -246,17 +264,21 @@ const trackParser = releaseEditor.trackParser = {
             }
         }
 
-        // MBS-7719: make sure the "Reuse previous recordings" button is
-        // available for new tracks by saving any unset recordings onto the
-        // new track instances.
+        /*
+         * MBS-7719: make sure the "Reuse previous recordings" button is
```
Comment (yvanzo) [inline root/static/scripts/release-editor/trackParser.js]:
```suggestion
         * MBS-7719: Make sure the "Reuse previous recordings" button is
```

File: root/static/scripts/common/MB.js [hunk 16]
```diff
@@ -1,20 +1,9 @@
...
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program; if not, write to the Free Software
- * Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+ * Licensed under the GPL version 2, or (at your option) any later version:
+ * http://www.gnu.org/licenses/gpl-2.0.txt
```
Comment (yvanzo) [inline root/static/scripts/common/MB.js]: This is not the newest blurb, see https://musicbrainz.org/doc/Development/Server_Coding_Style#Templates which occurs in half of JavaScript files already. Should be:

```
 * Copyright (C) 2009 Olivier Charles
 *
 * This file is part of MusicBrainz, the open internet music database,
 * and is licensed under the GPL version 2, or (at your option) any
 * later version: http://www.gnu.org/licenses/gpl-2.0.txt
```

Comment (reosarevok) [reply root/static/scripts/common/MB.js]: Dammit! :) I'll fix it today.

**Comment by reosarevok** [comment]
@yvanzo let me know I was in fact not using the latest copyright blurb for the second commit, so I changed it to update all the old blurbs to (I think???) the newest ones :)

**Pull request merged**